egs/heroico/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
```

Comment:
I have just noticed this file -- what recipe did you use as the starting code?
We do these things now more effectively -- unless it's a lot of modifications, could you use 
mini_librispeech/s5/local/nnet3/run_ivector_common.sh instead of this?
We didn't upgrade the older recipes, yet, but we are trying to make the new ones to use that.